datahub-web-react/src/app/ingestV2/executions/utils.ts

```diff
@@ -46,7 +46,7 @@ export const getExecutionRequestStatusDisplayText = (status: string) => {
         (status === EXECUTION_REQUEST_STATUS_RUNNING && 'Running') ||
         (status === EXECUTION_REQUEST_STATUS_SUCCESS && 'Success') ||
         (status === EXECUTION_REQUEST_STATUS_SUCCEEDED_WITH_WARNINGS && 'Success') ||
-        (status === EXECUTION_REQUEST_STATUS_FAILURE && 'Failed') ||
```

Comment:
I don't think this change in copy was intended- let's revert this.

Reply:
The design is explicitly changing the wording. Please check figma